tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -490,6 +490,90 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
     }
   }
 
+  "Document EDEK Encrypt" should {
```

Comment:
what can we assert about the edeks themselves? Maybe that they exist and are non zero length?

Reply:
We should be able to assert the length of them. If you encrypt to a few people you should verify the number of edeks that come back.

Reply:
Since they are bytes, it's not straightforward to make this assertion. We can assert about relative lengths, I guess. 1 edek should be larger than 2 edeks, etc.

Reply:
Right, forgot they came back as protobuf.

Reply:
Added assertions that the edeks are not empty